src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/services/HelperApi.kt

```diff
@@ -5,7 +5,7 @@ import org.slf4j.LoggerFactory
 
 val log: Logger = LoggerFactory.getLogger("HelperApi")
 
-suspend fun <T> doApiCallWithSingleRetry(api: suspend () -> T) = runCatching {
+suspend fun <T> doApiCallWithRetries(api: suspend () -> T) = runCatching {
```

Comment:
I know this works but isn't it something we should be doing at the web client level? https://www.baeldung.com/spring-webflux-retry#1-usingretry